docs/en/connector-v2/source/Doris.md

```diff
@@ -0,0 +1,121 @@
+# Doris
+
+> Doris source connector
+
+## Description
+
+Used to read data from Doris.
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [x] [schema projection](../../concept/connector-v2-features.md)
+- [x] [parallelism](../../concept/connector-v2-features.md)
+- [x] [support user-defined split](../../concept/connector-v2-features.md)
+
+## Options
+
+|               name               |  type  | required | default value |
+|----------------------------------|--------|----------|---------------|
+| fenodes                          | string | yes      | -             |
+| username                         | string | yes      | -             |
+| password                         | string | yes      | -             |
+| table.identifier                 | string | yes      | -             |
+| schema                           | config | yes      | -             |
```

Comment:
I don't understand what the purpose of the scheme parameter is? Doris is a data source with a table schema, and connectors can obtain the table structure information of the tables that need to be read through Doris connections. If the schema is only used to specify the `schema projection` feature, I believe that using the schema is not a good choice in this case, because the schema needs to configure fields and their type information, which is redundant for the `schema projection`. An array type parameter that only needs to configure field columns would be more suitable.

Reply:
Did you mean  `fields = [f1,f2,f3,...]`?

Reply:
> Did you mean `fields = [f1,f2,f3,...]`?

Yes.